docs/conf.py

```diff
@@ -1,8 +1,11 @@
 # -*- coding: utf-8 -*-
-import sys, os, pkginfo, datetime
+import datetime
```

Comment:
They make for cleaner diffs when you add or remove imports, since only the modified import appears in the diff. (it's possible to make exceptions to pep8 if you still prefer the current way)